sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5986,9 +5986,10 @@ pi_result piEventGetProfilingInfo(pi_event Event, pi_profiling_info ParamName,
     ContextEndTime *= ZeTimerResolution;
     return ReturnValue(ContextEndTime);
   }
+  // TODO: Remove this and other related code for setting or getting
+  // queued/submit time
```

Comment:
please explain the comment

Reply:
why don't remove them here then?
also I am worried that plugins now have no visibility into event's queue/submit time

Reply:
I'll remove them. 


> also I am worried that plugins now have no visibility into event's queue/submit time

An event's queue/submit time would now be calculated when a command is submitted to SYCL RT, not necessarily plugin; so I don't know how having visibility for it would be useful. 
If by queue/submit time you mean when a kernel is submitted to a device it-self, than maybe I shouldn't remove this code.

Reply:
> An event's queue/submit time would now be calculated when a command is submitted to SYCL RT, not necessarily plugin; so I don't know how having visibility for it would be useful.
> If by queue/submit time you mean when a kernel is submitted to a device it-self, than maybe I shouldn't remove this code.

Yeah, I think we should preserve the handling of PI_PROFILING_INFO_COMMAND_SUBMIT/PI_PROFILING_INFO_COMMAND_QUEUEED with the meaning of when commands were submitted to the device. [Even if not used today by SYCL RT]